src/js/tech/html5.js

```diff
@@ -34,7 +34,9 @@ class Html5 extends Tech {
   constructor(options, ready) {
     super(options, ready);
 
-    this.setupSourcesetHandling_();
+    if (options.enableSourceset) {
```

Comment:
I am wondering what you think about moving the `featuresSourceSet` check here instead of inside of the function? seems closer to how our other feature checks work.

Reply:
all the features are set more or less statically, then there's a secondary check as to whether we want to use the feature or not.

Reply:
ok